src/codecatalyst/auth.ts

```diff
@@ -106,18 +106,18 @@ export class CodeCatalystAuthenticationProvider {
 
         const conn = (await this.auth.listConnections()).find(isBuilderIdConnection)
         const isNewUser = conn === undefined
-        const okItem: vscode.MessageItem = { title: localizedText.ok }
+        const proceedItem: vscode.MessageItem = { title: localizedText.proceed }
         const cancelItem: vscode.MessageItem = { title: localizedText.cancel, isCloseAffordance: true }
 
         if (isNewUser || !isValidCodeCatalystConnection(conn)) {
             // TODO: change to `satisfies` on TS 4.9
             telemetry.record({ codecatalyst_connectionFlow: isNewUser ? 'Create' : 'Upgrade' } as ConnectionFlowEvent)
 
-            const message = isNewUser
-                ? 'CodeCatalyst requires an AWS Builder ID connection. Creating a connection opens your browser to login.\n\n Create one now?'
-                : 'Your AWS Builder ID connection does not have access to CodeCatalyst. Upgrading the connection requires another login.\n\n Upgrade now?'
-            const resp = await vscode.window.showInformationMessage(message, { modal: true }, okItem, cancelItem)
-            if (resp !== okItem) {
+            const message =
+                'The AWS Toolkit for VSCode requires a connection for CodeCatalyst to begin.\n\n Proceed to the browser to allow access?'
```

Comment:
looks for usages of `getIdeProperties().company` to see how we normally refer to the Toolkit in messages. Needs to be "Amazon", not "AWS", in some regions, thus `getIdeProperties().company` instead of "AWS".

If there's a chance this may show in non-vscode contexts such as Cloud9, I'd drop the mention of that also. If for some reason it's necessary to mention "for VSCode" here, could use `getIdeProperties().shortName`